src/sentry/integrations/github/client.py

```diff
@@ -224,14 +234,18 @@ def _populate_trees_process_error(self, error: ApiError, extra: Dict[str, str])
             logger.warning(f"Github has blocked the repository. {msg}", extra=extra)
         elif txt == "Server Error":
```

Comment:
Sure, but I'll admit I'm not totally sure what we'd be testing. Though I can definitely do them, straight up unit tests of that function (`_populate_trees_process_error`) feel a little silly, because either they're just checking that a given incoming error string results in the correct logged string, or they're testing that it returns `True` or `False` correctly, neither of which are based on any logic beyond what we'd have to bake into the test to even write it. But if that's not what you want to test, what exactly _do_ you want to test?

That said, based on the GH outage this morning landing us in the generic "investigate this error" branch, I do think we have another case to add to our series of `if`s (though that really should be in a separate PR).